huaweicloud/services/cce/resource_huaweicloud_cce_node.go

```diff
@@ -429,10 +432,77 @@ func ResourceNode() *schema.Resource {
 			"auto_pay":      common.SchemaAutoPay(nil),
 
 			"extend_param": {
-				Type:     schema.TypeMap,
+				Type:        schema.TypeMap,
+				Optional:    true,
+				ForceNew:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: "schema: Deprecated",
+			},
+			"extend_params": {
```

Comment:
can we extract the schema as a function? then it can be used by cce_node and cce_node_pool

Reply:
sure, move the extend_params schema and related functions to cce/common.go